src/main/java/com/example/springboot/entity/Validator.java

```diff
@@ -0,0 +1,44 @@
+package com.example.springboot.entity;
+
+import com.example.springboot.constants.ResultInfoConstants;
+import com.example.springboot.exception.InvalidException;
+import com.example.springboot.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class Validator {
+    private final UserRepository userRepository;
+
+    public final boolean validateId(Long userId) {
+        int count = userRepository.checkUserExistsById(userId);
+        if (count == 0) {
+            return false;
+        } else {
+            return true;
+        }
```

Comment:
same

Reply:
but do we need this method?

Reply:
Otp is generated and validated based on UserId.
So to check whether that Id exists in UserTable I i used this method.